server/bce/migrations/0001_setup_bce_database.py

```diff
@@ -7,32 +7,57 @@
 
 class Migration(migrations.Migration):
 
+    replaces = [
+        ("bce", "0001_initial"),
+        ("bce", "0002_auto_20181119_1632"),
+        ("bce", "0003_auto_20181119_1700"),
+        ("bce", "0004_auto_20181119_1702"),
+        ("bce", "0005_auto_20181121_1622"),
+        ("bce", "0006_auto_20200301_1937"),
+    ]
+
     initial = True
 
-    dependencies = [migrations.swappable_dependency(settings.AUTH_USER_MODEL)]
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
what's with the comma? is there a missing parameter/argument?

Reply:
https://recology.info/2019/02/trailing-commas/